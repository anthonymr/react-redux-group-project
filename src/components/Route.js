import { NavLink } from 'react-router-dom';

function Route() {
  return (
    <ul>
      <li>
        <NavLink to="/">Rockets</NavLink>
      </li>
      <li>
        <NavLink to="missions">Missions</NavLink>
      </li>
    </ul>
  );
}
export default Route;
