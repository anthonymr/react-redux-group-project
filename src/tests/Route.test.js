import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Route from '../components/Route';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Route />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
