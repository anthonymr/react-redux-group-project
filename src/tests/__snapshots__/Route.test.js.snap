// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul>
  <li>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
    >
      Rockets
    </a>
  </li>
  <li>
    <a
      className=""
      href="/missions"
      onClick={[Function]}
    >
      Missions
    </a>
  </li>
  <li>
    |
  </li>
  <li>
    <a
      className=""
      href="/myprofile"
      onClick={[Function]}
    >
      My profile
    </a>
  </li>
</ul>
`;
